microshift_release_notes/microshift-4-19-release-notes.adoc

```diff
@@ -204,3 +204,21 @@ See the latest images included with {microshift-short} by using the following in
 
 * xref:../microshift_updating/microshift-list-update-contents.adoc#microshift-get-rpm-release-info_microshift-list-update-contents[Listing the contents of the {microshift-short} RPM release package]
 * xref:../microshift_install_bootc/microshift-install-bootc-image.adoc#microshift-install-bootc-get-published-image_microshift-install-publish-bootc-image[Getting the published bootc image for MicroShift]
+
+[id="microshift-4-19-7-dp_{context}"]
+=== RHBA-2025:12745 - {microshift-short} 4.19.7 bug fix and update advisory
+
+Issued: 5 August 2025
+
+{product-title} release 4.19.7 is now available. Bug fixes and enhancements are listed in the link:https://access.redhat.com/errata/RHBA-2025:12745[RHBA-2025:12745] advisory. Release notes for bug fixes and enhancements are provided in this documentation. The images that are included in the update are provided by the {OCP} link:https://access.redhat.com/errata/RHSA-2025:12341[RHSA-2025:12341] advisory.
+
+See the latest images included with {microshift-short} by using the following instructions:
+
+* xref:../microshift_updating/microshift-list-update-contents.adoc#microshift-get-rpm-release-info_microshift-list-update-contents[Listing the contents of the {microshift-short} RPM release package]
+* xref:../microshift_install_bootc/microshift-install-bootc-image.adoc#microshift-install-bootc-get-published-image_microshift-install-publish-bootc-image[Getting the published bootc image for MicroShift]
+
+==== Bug fixes
+
+* Before this update, a bare-metal node reboot caused the deployment progress deadline to be surpassed. This resulted in the greenboot health check failing with a false-positive error. With this release, the `ProgressDeadlineExceeded` condition for deployments is removed to allow time for reboots that exceed the default time limit. Now, the full timeout duration is provided for the deployment to become ready, reducing false-positive greenboot health check errors. https://issues.redhat.com/browse/OCPBUGS-59301[OCPBUGS-59301]
+
+* Before this update, the excessive polling frequency for the cloud pull secret in cluster telemetry caused telemetry failure messages to flood journald logs. The log flooding negatively impacted system performance. With this release, the telemetry collection interval is updated from 1 second to 30 seconds. As a result, telemetry logs no longer flood, leading cleaner logs in cases of misconfiguration. https://issues.redhat.com/browse/OCPBUGS-59240[OCPBUGS-59240]
```

Comment:
Not sure if  we need this.`leading cleaner logs in cases of misconfiguration`  also i would say change `As a result, telemetry logs no longer flood` to As a result flooding has been minimized.